src/Microsoft.IdentityModel.Abstractions/Result.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+#nullable enable
+namespace Microsoft.IdentityModel.Abstractions
+{
+    /// <summary>
+    /// Represents a result that can be either successful or unsuccessful.
+    /// </summary>
+    /// <typeparam name="TResult"></typeparam>
+    /// <typeparam name="TError"></typeparam>
+    public readonly struct Result<TResult, TError> : IEquatable<Result<TResult, TError>>
```

Comment:
Since this is not abstract, the class should not be in M.IM.Abstractions.
Need to move to M.IM.Tokens.

Reply:
Moved